app/models/legislator.rb

```diff
@@ -0,0 +1,54 @@
+class Legislator < ActiveRecord::Base
+  belongs_to :district
+  belongs_to :state
+
+  validates :bioguide_id, presence: true, uniqueness: true
+
+  scope :senate, -> { where(chamber: 'senate') }
+  scope :house,  -> { where(chamber: 'house') }
+
+  def self.fetch(bioguide_id: nil, district: nil, state: nil, senate_class: nil)
+    params = sunlight_params(bioguide_id, district, state, senate_class) or return
+
+    results = Integration::Sunlight.get_legislator(params)
+
+    if stats = results['legislator']
+      bioguide_id = stats.delete('bioguide_id')
+      stats = replace_state_and_district(stats)
+      create_with(stats).find_or_create_by(bioguide_id: bioguide_id)
+    end
+  end
+
+  def update_stats
+    results = Integration::Sunlight.get_legislator(bioguide_id: bioguide_id)
+    if stats = results['legislator']
+      stats = self.class.replace_state_and_district(stats)
+      update(stats)
+    end
+  end
+
+  private
+
+  def self.sunlight_params(bioguide_id, district, state, senate_class)
```

Comment:
I'd consider making `sunlight_params` a helper method in the `Integration::Sunlight` class.  Then you might have a public `Integration::Sunlight.query_legislator_by_params({})` and have it call a `Integration::Sunlight.parse_params` before passing to `Integration::Sunlight.get_legislator`.

Other notes:
* I'm not sure these all need to be class methods, but no biggy.
* we should set up a method to fetch_all legislators. It would be good if it could parse a few at a time, but I can see how you need to replace them with the target IDs.
* `replace_state_and_district`, `update_stats` and `stats` are a bit vague.  Maybe something like `find_location_associations` `location_associations` - I'm not sure what's most appropriate.